packages/core/src/components/Modal/index.tsx

```diff
@@ -60,7 +60,7 @@ export default withStyles(({ color, unit }) => ({
     display: 'flex',
     justifyContent: 'center',
     minHeight: '100%',
-    padding: unit * 8,
```

Comment:
this large padding was only noticeable on small screens. smaller now that modal widths are fixed.